03_launch_mgmt_cluster.sh

```diff
@@ -29,13 +29,22 @@ M3PATH="${M3PATH:-${GOPATH}/src/github.com/metal3-io}"
 BMOPATH="${BMOPATH:-${M3PATH}/baremetal-operator}"
 RUN_LOCAL_IRONIC_SCRIPT="${BMOPATH}/tools/run_local_ironic.sh"
 CAPM3PATH="${CAPM3PATH:-${M3PATH}/cluster-api-provider-metal3}"
+CAPM3REPO="${CAPM3REPO:-https://github.com/metal3-io/cluster-api-provider-metal3.git}"
```

Comment:
you do not have to add the .git at the end. If you do not provide it, then you can use CAPM3REPO as the base url for CAPM3RELEASEPATH

Reply:
Changed completely url parsing